### ISSUE TITLE
Favicon not showing for some pages

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#3055]
The favicon is working for some pages, but not others when opening a new tab. If going to a page from another page where he favicon has loaded, then it works.: <img width="888" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOHRBQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ee57fee781c18237cbb499f7f134ec20787aad1b/image.png" alt="image.png" />

https://onrr.gov/about/foia
https://onrr.gov/about/contact
https://onrr.gov/about/office-locations

**Comment by greensomes** [comment]
@eelzi-ONRR , let's touch base on this Monday. I was unable to reproduce the issue in Safari and Chrome on macOS.

**Comment by greensomes** [comment]
@eelzi-ONRR , I observed this happening in dev and preview. I've applied a change that, I believe, resolves the issue. The path to the favicon file in our index.html file was incorrect. The steps I took to reproduce in [dev](https://dev-onrr-frontend.app.cloud.gov)/prod and validate in [preview](https://preview-onrr-frontend.app.cloud.gov) is:

- Navigate to the home page. (favicon appears)
- Click on the "Contacts" link in the header. (favicon sometimes appears)
- Click on the header link in the "Leases & Agreements" card. (favicon was not appearing)
